docs/COMPATIBILITY.md

```
@@ -14,150 +14,137 @@ Semantic MediaWiki may also work with more recent versions of PHP and MediaWiki,
-        <td></td>
-    </tr>
-    <tr>
-        <th><a href="https://www.semantic-mediawiki.org/wiki/Semantic_MediaWiki_1.9.0">1.9.x</a></th>
-        <td>Obsolete release</td>
-        <td>2014-01-03</td>
-        <td>2014-04-18</td>
-        <td>5.3.2 - 5.6.x</td>
-        <td>1.19.0 - 1.22.x</td>
-        <td></td>
-    </tr>
+	<tr>
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)